src/main/kotlin/blackjack/domain/Dealer.kt

```diff
@@ -1,10 +1,60 @@
 package blackjack.domain
 
+import blackjack.domain.enums.Condition
+import blackjack.domain.enums.MatchResult
+
 class Dealer(
-    val deck: Deck
-) {
+    name: String = "딜러",
+    cards: Cards,
+    condition: Condition = Condition.STAY,
+    val deck: Deck,
+) : Participant(name, cards, condition) {
+
+    init {
+        val score = cards.calculateScore()
+        initCondition(score)
+    }
+
+    private fun initCondition(score: Score) {
+        if (score.compareTo(Score.STANDARD_CARD_SCORE) == -1 || score.compareTo(Score.STANDARD_CARD_SCORE) == 0) {
+            this.condition = Condition.PLAY
+        }
+    }
 
-    fun drawCardsFromDeck(count: Int): Cards {
+    override fun hit(card: Card) {
+        super.hit(card)
+
+        val score = cards.calculateScore()
+        changeCondition(score)
+    }
+
+    private fun changeCondition(score: Score) {
+        if (score.compareTo(Score.STANDARD_CARD_SCORE) == 1) {
+            this.condition = Condition.STAY
+        } else if (score.compareTo(Score.BLACK_JACK_SCORE) == 1) {
+            this.condition = Condition.BUST
+        }
```

Comment:
오버라이딩한 compareTo 함수를 infix 함수로 선언하면 아래와 같이 표현할 수 있습니다🙂
```suggestion
        if(score == Score.STANDARD_CARD_SCORE) {
            this.condition = Condition.BLACKJACK
        }
        if(score == Score.BLACK_JACK_SCORE) {
            this.condition = Condition.BLACKJACK
        }
```

Reply:
점수 비교 로직 수정하였습니다!